stylesheets/application.css

```diff
@@ -910,4 +910,11 @@ a.page__heading {
     .nav--v__link--footer:last-child {
       margin-bottom: 36px; } }
 
+.t-body .t-img {
```

Comment:
will add this as scss in seperate PR. also, need to fix this https://github.com/rubygems/guides/commit/c1f6b2ef710bb1b203aa100cdafd52244fd2d49b#diff-33530e60b01f198d2a5b4e0160101137